File: backend/src/main/java/model/Customer.java
package model;

public class Customer {
    private int id;
    private String name, eMail;

    public Customer(int id, String name, String eMail) {
        this.id = id;
        this.name = name;
        this.eMail = eMail;
    }

    public int getId() {
        return id;
    }

    public String getName() {
        return name;
    }

    public String geteMail() {
        return eMail;
    }
}

File: frontend/web/src/app/model/customer.ts
export class Customer {

  public id: number;
  public name: string;
  public eMail: string;

  constructor(values: Object = {}) {
    if (!values)
      return null;
    Object.assign(this, values);
  }

  static create(json: any) {
    if (json)
      return new Customer(json);
    else
      return null;
  }
}
